src/main/java/io/confluent/connect/jdbc/source/DataConverter.java

```diff
@@ -54,7 +54,13 @@ protected Calendar initialValue() {
     }
   };
 
-  public static Schema convertSchema(String tableName, ResultSetMetaData metadata)
+  private final boolean numericPrecisionMapping;
+
+  DataConverter(JdbcSourceTaskConfig config) {
+    this.numericPrecisionMapping = config.getBoolean(JdbcSourceConnectorConfig.NUMERIC_PRECISION_MAPPING_CONFIG);
```

Comment:
I'd be in favor of leaving this as a class with purely static methods, and making the `numericPrecisionMapping` a boolean argument for the methods that depend on it.